### DISCUSSION TITLE
Gradients?

### DISCUSSION THREAD

**Discussion by sdfg2** [piroor/treestyletab#3224] [Q&A] [answered]
Is `linear-gradient` supported?

```
tab-item.active {
  background-image: linear-gradient(to-right, #7040ff, #05b5ff) !important;
}
```

doesn't seem to work.

**Comment by irvinm** [comment, marked as answer]
You don't need the "-" for "to right" and I think you want to apply it to ".background":

```
tab-item.active .background {
  background: linear-gradient(to right, #7040ff, #05b5ff);
}
```

![image](https://user-images.githubusercontent.com/979729/194715406-7dfefc77-9277-4dda-8829-465f89df270a.png)

**Reply by sdfg2** [reply]
Ah, don't need background, just `to right` not `to-right`.
